src/js/stores/MesosStateStore.js

```diff
@@ -28,12 +24,10 @@ import MesosStateUtil from "../utils/MesosStateUtil";
 import pipe from "../utils/pipe";
 import { linearBackoff } from "../utils/rxjsUtils";
 import { MesosStreamType } from "../core/MesosStream";
+import { MesosMasterRequestType } from "../core/MesosMasterRequest";
```

Comment:
Shall we consider refactoring the last place in OverviewDetailTab.js to use request from container and replaying the response instead of polling?